packages/server/test/cache_helper.ts

```diff
@@ -0,0 +1,14 @@
+import { CYPRESS_CONFIG_FILES } from '../lib/configFiles'
+
+/**
+ * Since we load the cypress.json via `require`,
+ * we need to clear the `require.cache` before/after some tests
+ * to ensure we are not using a cached configuration file.
+ */
```

Comment:
any reason we don't use `fs.read` if it's JSON instead and skip this wart?

Reply:
The only reason is the code in lib/util/settings is made simpler.

Reply:
Can you update it to use `fs.read` in the case of `.json` and remove this cache deletion thing? it seems like with the choice of clearing require cache vs. using `fs.read`, the latter is more understandable/maintainable. Unless I'm misunderstanding and it is a big lift to change this now.

Reply:
I optimized things a little in this commit 47d36f6 but left the cache clean process. 
I wonder if I should remove the process everywhere it is unnecessary fo test speed.

... will try that and see if there is a diff.

Reply:
I am hoping to keep as many of those tests as possible when we deprecate `cypress.json`. This will mean dealing with the cache will be inevitable. I would rather start now since we need for some tests.

Reply:
Literally 100% missed this was a test util. Carry on :smile: :gun: